api/src/test/java/org/openmrs/api/db/hibernate/HibernateOrderDAOTest.java

```diff
@@ -109,5 +116,62 @@ public void getOrderGroupsByPatient_shouldGetOrderGroupsGivenPatient() {
 		Patient existingPatient = Context.getPatientService().getPatient(8);
 		List<OrderGroup> ordergroups = Context.getOrderService().getOrderGroupsByPatient(existingPatient);
 		assertEquals(1, ordergroups.size());
+		
+	}
+	
+	/**
+	 * @see {@link HibernateOrderDAO#getAllOrderGroupAttributeTypes()}
+	 * @throws Exception
+	 */
+    @Test
+	public void getAllOrderGroupAttributeTypes_shouldGetAllOrderGroupAttributeTypes(){
+		List<OrderGroupAttributeType> orderGroupAttributeTypes = dao.getAllOrderGroupAttributeTypes();
+		assertEquals(orderGroupAttributeTypes.size(), 4);
+	}
+	
+	/**
+	 * @see {@link HibernateOrderDAO#getOrderGroupAttributeTypeByUuid(String)}
+	 * @throws Exception
+	 */
+	@Test
+	public void getOrderGroupAttributeType_shouldGetOrderGroupAttributeTypeGivenUuid(){
+    	OrderGroupAttributeType newOrderGroupAttributeType = dao.getOrderGroupAttributeTypeByUuid("9cf1bce0-d18e-11ea-87d0-0242ac130003");
+		assertEquals("Bacteriology", newOrderGroupAttributeType.getName());
+	}
+	
+	/**
+	 * @see {@link HibernateOrderDAO#getOrderGroupAttributeType(Integer)}
+	 * @throws Exception
+	 */
+	@Test
+	public void getOrderGroupAttributeType_shouldReturnOrderGroupAttributeTypeGivenIntegerId(){
+		OrderGroupAttributeType newOrderGroupAttributeType = dao.getOrderGroupAttributeType(4);
+		assertEquals("ECG", newOrderGroupAttributeType.getName());
+	}
+	
+	/**
+	 * @see {@link HibernateOrderDAO#getOrderGroupAttributeTypeByName(String)}
+	 * @throws Exception
+	 */
+    @Test
+    public void getOrderGroupAttributeTypeByName_shouldGetOrderGroupAttributeTypeByName(){
+		final String NAME = "ECG";
+		OrderGroupAttributeType newOrderGroupAttributeType = dao.getOrderGroupAttributeTypeByName(NAME);
+		assertEquals(NAME, newOrderGroupAttributeType.getName());
+		assertEquals(4, newOrderGroupAttributeType.getId());
+		assertEquals("9cf1bdb2-d18e-11ea-87d0-0242ac130003", newOrderGroupAttributeType.getUuid());
+    }
+    
+    /**
+	 * @see {@link HibernateOrderDAO#deleteOrderGroupAttributeType(OrderGroupAttributeType)}
+	 * @throws Exception
+	 */
+    @Test
+	public void deleteOrderGroupAttributeType_shouldDeleteOrderGroupAttributeTypeFromDatabase(){
+    	final String UUID1 = "9cf1bdb2-d18e-11ea-87d0-0242ac130003";
```

Comment:
The TABBING above.
Do you run mvn clean install? I remember seeing it automatically format the code.

Reply:
Did you see the above?

Reply:
And why did you append 1 to the constant name?

Reply:
I have removed the 1

@dkayiwa  this is the indenting which i have locally
![idea1](https://user-images.githubusercontent.com/16301386/92429364-55810d80-f19a-11ea-850d-4fcf7a0c2f95.png)